es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Chuleta/Acordeón de Deep Learning
 
 <br>
 
 **2. Neural Networks**
 
-&#10230;
+&#10230; Redes Neuronales
 
 <br>
 
 **3. Neural networks are a class of models that are built with layers. Commonly used types of neural networks include convolutional and recurrent neural networks.**
 
-&#10230;
+&#10230; Las redes neuronales son una clase de modelos construidos a base de capas. Los tipos más utilizados de redes neuronales incluyen las redes neuronales convolucionales y las redes neuronales recurrentes.
 
 <br>
 
 **4. Architecture ― The vocabulary around neural networks architectures is described in the figure below:**
 
-&#10230;
+&#10230; Arquitectura - el vocabulario en torno a arquitecturas de redes neuronales se describe en la figura siguiente:
```

Comment:
Suggestion, change this to:

&#10230; Arquitectura - el vocabulario en torno a la arquitectura de las redes neuronales se describe en la siguiente figura:

Reply:
Also, as you can see in the [French translation](https://github.com/shervinea/cheatsheet-translation/blob/master/fr/cheatsheet-deep-learning.md) (by the authors of the repo). The try to include the English term between parenthesis when is common to find it even in Spanish literature. For example:

> Las redes neuronales son una clase de modelos construidos a base de capas. Los tipos más utilizados de redes neuronales incluyen las redes neuronales convolucionales y las redes neuronales recurrentes.

To

> Las redes neuronales (neural networks en inglés) son una clase de modelos construidos a base de capas. Los tipos más utilizados de redes neuronales incluyen las redes neuronales convolucionales (convolutional neural networks en inglés) y las redes neuronales recurrentes (recurrent neural networks en inglés).

Of course, we would have to discuss which terms should we include or not. But I think you can see the point.